src/js/components/ChatControl/ChatControl.test.js

```diff
@@ -24,13 +23,15 @@ describe('ChatControl', () => {
         avatar: foo,
       },
     initialTimeStamp: '2000-01-01T00:00:00',
-    messageLog: [
-      {
-        name: foo,
-        timeStamp: fooTimeStamp,
-        body: foo,
-      },
-    ],
+    [foo]: {
```

Comment:
What does putting this in square brackets do?

Reply:
It is so I can use the value of foo as the key for that object